### PR TITLE
Update installation-creating-azure-worker.adoc

### PR DESCRIPTION
**Pull request by nikhilprajapati-world**
As per doc bug raised https://issues.redhat.com/browse/OCPBUGS-29724, I prpopse to mention in the docs about the parameters to set the  VM size when using the templates,

Parameters to set the VM size when using the templates which using architecture as x86_64

$ az deployment group create -g ${RESOURCE_GROUP} \
  --template-file "<installation_directory>/06_workers.json" \
  --parameters workerIgnition="${WORKER_IGNITION}" \ 
  --parameters baseName="${INFRA_ID}" \
  --parameters nodeVMSize="Standard_D4s_v3"

Parameters to set the VM size when using the templates which using architecture as ARM/ARCH64

$ az deployment group create -g ${RESOURCE_GROUP} \
  --template-file "<installation_directory>/06_workers.json" \
  --parameters workerIgnition="${WORKER_IGNITION}" \ 
  --parameters baseName="${INFRA_ID}" \
  --parameters nodeVMSize="Standard_D4ps_v5"

Version(s):

Issue:

Link to docs preview:

https://docs.openshift.com/container-platform/4.15/installing/installing_azure/installing-azure-user-infra.html#installation-creating-azure-worker_installing-azure-user-infra

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by mjpytlak — COMMENTED** [review]
@nikhilprajapati-world Some feedback based on a peer review.

File: modules/installation-creating-azure-worker.adoc [hunk 1]
```diff
@@ -2,23 +2,17 @@
...
 :ash:
 :cp: Azure Stack Hub
 endif::[]
-ifeval::["{context}" == "installing-restricted-networks-azure-user-provisioned"]
-:azure:
-:cp: Azure
-endif::[]
 
-:_mod-docs-content-type: PROCEDURE
+:_content-type: PROCEDURE
```
Comment (mjpytlak) [inline modules/installation-creating-azure-worker.adoc:15]: The meta data should not be updated. `_mod-docs-content-type: PROCEDURE` is the correct meta data.

**Comment by mjpytlak** [comment]
@kelbrown20 I see that you are assigned to https://issues.redhat.com/browse/OCPBUGS-29724. FYI.

**Comment by kelbrown20** [comment]
Updates were already made in this PR: https://github.com/openshift/openshift-docs/pull/72310

**Comment by mjpytlak** [comment]
Closing at @kelbrown20 request.

**Pull request closed without merging**